app/main.py

```diff
@@ -1 +1,44 @@
-#  write your code here
+class Animal:
+
+    def __init__(self, name: str, appetite: int, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+
+    def __init__(self, name: str, is_hungry=True):
+
+        super().__init__(name, appetite=3)
+        self.is_hungry = is_hungry
```

Comment:
Your parent's init can set `self.is_hungry`, why do you duplicate yourself?

Reply:
How I can use non positional argument after positinal argument?